database/migrations/2017_05_12_111922_add_build_propagation_history_table.php

```diff
@@ -0,0 +1,32 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Support\Facades\Schema;
+
+class AddBuildPropagationHistoryTable extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::create('osu_build_propagation_history', function ($table) {
+            $table->increments('id');
+            $table->integer('stream_id')->unsigned();
```

Comment:
this stores data from all builds being online at any given moment, so I don't think it would make sense to associate a build version to it.